flink-end-to-end-tests/test-scripts/test_streaming_classloader.sh

```diff
@@ -31,6 +31,22 @@ echo "classloader.resolve-order: parent-first" >> "$FLINK_DIR/conf/flink-conf.ya
 echo "Moving fake LibPackage.jar from end-to-end tests to lib/"
 cp ${END_TO_END_DIR}/flink-parent-child-classloading-test-lib-package/target/LibPackage.jar ${FLINK_DIR}/lib/
 
+function classloader_cleanup() {
+  # don't call ourselves again for another signal interruption
+  trap "exit -1" INT
+  # don't call ourselves again for normal exit
+  trap "" EXIT
+
+  stop_cluster
```

Comment:
I think the stopping is also done by the runner script. But I guess it doesn't harm.

Reply:
I left it in as a safety, yes.